pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -160,10 +178,14 @@ func NewInstallerController(
 		podsGetter:       podsGetter,
 		eventRecorder:    eventRecorder.WithComponentSuffix("installer-controller"),
 		now:              time.Now,
+		startupMonitorEnabled: func() (bool, error) {
+			return false, nil
+		},
 
 		installerPodImageFn: getInstallerPodImageFromEnv,
 		clock:               clock.RealClock{},
 		installerBackOff:    backOffDuration(10*time.Second, 1.5, 10*time.Minute),
+		fallbackBackOff:     backOffDuration(10*time.Minute, 1.5, 2*time.Hour),
```

Comment:
that means we are going to retry `5` times in the first hour, hope that a possible outage is acceptable.

Reply:
better factor 2 ?

0:10, 0:20, 0:40, 1:20, 2:00 ?

Reply:
done